TestApp/app/screens/AppCenterScreen.js

```diff
@@ -233,6 +233,9 @@ export default class AppCenterScreen extends Component {
                   value: 'userId',
                   onChange: async (userId) => {
                     this.setState({ userId });
+                  },
+                  onSubmit: async () => {
+                    const userId = this.state.userId.length === 0 ? null : this.state.userId;
```

Comment:
Add a comment to explain why converting empty string to `null`. It's because 1DS `setUserId` API allows `null` but not empty string as `userId`.